app/components/meetups/mainstage/Mainstage.js

```diff
@@ -0,0 +1,50 @@
+import { useState } from 'react';
+import { Collapse } from 'react-bootstrap';
+import styles from '../../../styles/meetup.module.css';
+import dynamic from 'next/dynamic';
+import { useMediaQuery } from '@rocket.chat/fuselage-hooks';
+import { ChatToolBar } from './Toolbar';
+import { YoutubeVideoComponent } from './VideoEmbedComponent';
+
+const RCComponent = dynamic(() => import('rc-component-react').then((mod) => mod.RCComponent), {
+  ssr: false,
+});
+
+export const MeetupMainstage = () => {
+  const isSmallScreen = useMediaQuery('(max-width: 790px)');
+  const [open, setOpen] = useState(isSmallScreen);
+
+  return (
+    <div className={styles.mainstage_root}>
+      <YoutubeVideoComponent
+        videoId={'E7wJTI-1dvQ'}
```

Comment:
It would be better if we could have a button near the Top of the video, something along "Change Video". On clicking, a modal will open and ask for a new Youtube link, then on submitting this modal form, the embedded video should change to the submitted YT video link.

Thank you!

Reply:
@Sing-Li suggested that the parameters like youtubelink,channel,etc could only `be changed only via a rebuild and redeploy -- and only by the community builder who owns the site`.